readme.md

```diff
@@ -30,16 +30,27 @@ The output is a JSON object in the form:
 
 ```json
 {
-    "bundle_name": {
-        "asset_kind": "/public/path/to/asset"
-    }
+    "chunks": {
+      "bundle_name": {
+          "asset_kind": "/public/path/to/asset"
+      }
+    },
+    "images": [{
+        "name": "hello.svg",
+        "path": "/public/path/to/asset/hello.svg"
+      }
+    ]
```

Comment:
> Just less data to iterate over when looking up a URL.
 
I see your point, but since we have `/\.(jpe?g|png|gif|svg)$/)` as default pattern, you'll have the same amount of data.

Let's name it:
- `assets` for the key.
` `assetsRegex` for the option.